README.md

```diff
@@ -294,7 +294,13 @@ The example below gives AAPL daily OHLCV data in a DataFrame format.
 import alpaca_trade_api as tradeapi
 
 api = tradeapi.REST()
+# all of these examples work
 aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from='2019-01-01', to='2019-02-01').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01'), to='2019-01-10').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01').date(), to='2019-01-10').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=pd.Timestamp('2019-01-01'), to='2019-01-10').df
+# timestamp should be in milliseconds
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01').timestamp()*1000, to='2019-01-10').df
```

Comment:
Also I would put `1546329600000` instead of dateutil...

Reply:
sure. I just wanted to show that we start always from the same thing. but sure